server/pages/index.jade

```diff
@@ -11,10 +11,8 @@ doctype html
 
 html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width' : '')
 	head
-		if helmetTitle
```

Comment:
> Done. Note that this means an initialReduxState will now be created and passed in the skeleton HTML also for non-isomorphic sites, but it's fairly small for non-iso sections.

Whatever is in `initialReduxState` is not picked up from the local state cache, so we need to be careful. This will just be `ui` and `documentHead`?